01-node-tutorial/answers/04-names.js

```diff
@@ -0,0 +1,5 @@
+const peter = 'peter';
+const sara = 'sara';
+const david = 'david';
+
```

Comment:
I'm assuming that you're just practicing variables and these look fine. Another option might be to create an array called `names` and we can export that `names` array.

Would look like:

```
const names = ['peter', 'sara', 'david'];

module.exports = { names }
```

then in your modules file, you can access them using this index number, i.e. `names[0]` to access 'peter'.